frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -1,84 +1,143 @@
+    position: 'relative',
+    display: 'inline-block',
+});
 export const StyledLink = styled(NavLink)({
     textDecoration: 'none',
-    color: colors.text,
+    color: 'white',
+    fontFamily: 'System-ui',
+    fontSize: '18px',
+    fontWeight: 700,
+    lineHeight: '28px',
+    textAlign: 'left',
+});
+
+const NavbarStyledLink = styled(NavLink)({
+    textDecoration: 'none',
+    color: 'white',
+    fontFamily: 'System-ui',
+    fontSize: '18px',
+    fontWeight: 700,
+    lineHeight: '28px',
+    textAlign: 'left',
+    position: 'relative',
     '&.active': {
-        color: colors.blue,
+        color: 'white',
     },
-})
-
-const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'))
-const AccountLinks = loadAsync('Account Links', () => import('./account-links'))
+    '&.active::after': {
+        content: '""',
+        position: 'absolute',
+        bottom: '-2px',
+        left: 0,
+        width: '100%',
+        height: '4px',
+        backgroundColor: '#0EE094',
+    },
+});
 
-export const TopNavBar: FCWOC<TopNavBarProps> = ({ children, className }) => {
-    const user = useCurrentUser()
-    const hideBanner = user.isResearcher
+const AdminLinks = loadAsync('Admin Links', () => import('./admin-links'));
+const AccountLinks = loadAsync('Account Links', () => import('./account-links'));
+export const TopNavBar: React.FC<TopNavBarProps> = ({ className }) => {
+    const user = useCurrentUser();
+    const hideBanner = user.isResearcher;
+    const isMobile = useIsMobileDevice();
 
     return (
-        <nav className={cx('navbar', 'navbar-light', className)}>
-            <div className="navbar-dark bg-dark py-1">
-                <Container>
-                    <Box justify="between" align="center" gap padding={{ vertical: 'default' }}>
-                        <NavbarLogoLink />
-                        {children}
-                        <Box gap="xlarge" align="center">
-                            <Link to="/studies" css={{ color: 'white', textDecoration: 'none', fontWeight: 'bold' }}>
-                                Dashboard
-                            </Link>
-                            <DesktopResearcherLinks />
-
+        <nav className={cx('navbar', 'navbar-light', className)} style={{ backgroundColor: colors.navy, color: 'white', height: '80px' }}>
+            <div style={{ height: '80px', backgroundColor: colors.navy }}>
+                <Container style={{ height: '80px' }}>
+                    <Box 
+                        justify="between"
+                        align="center"
+                        gap="0px"
+                        css={{
+                            width: '100%',
+                            height: '80px',
+                            opacity: 1,
+                        }}
+                    >
+                        <Box gap="medium" align="center">
+                            <NavbarLogoLink /> 
+                        </Box>
+                        <Box
```

Comment:
instead of using <Box> we can use mantine's components like <Group> or <Stack>. these have most if not all of the properties youre using here, we want to get rid of `<Box>` from `sundry` and `boxible` and this is a good chance to do it